src/portable/st/stm32_fsdev/dcd_stm32_fsdev.c

```diff
@@ -258,12 +258,8 @@ void dcd_int_enable (uint8_t rhport)
 {
   (void)rhport;
 #if defined(STM32F0)
-  NVIC_SetPriority(USB_IRQn, 0);
   NVIC_EnableIRQ(USB_IRQn);
 #elif defined(STM32F3)
-  NVIC_SetPriority(USB_HP_CAN_TX_IRQn, 0);
-  NVIC_SetPriority(USB_LP_CAN_RX0_IRQn, 0);
```

Comment:
The priority shouldn't have changed, so no need to reset it here.

Reply:
yes, the priority should be set by application/bsp to suite their need. E.g in my BLE work, the ble task take higher priority than usb. Some rtos such as freeRTOS reserved highest IRQ prio for kernel etc... So it is best to leave that to user.